swim/labels.go

```diff
@@ -145,6 +131,24 @@ func (lo LabelOptions) validateLabels(current map[string]string, additional map[
 	return nil
 }
 
+func countLabels(labels map[string]string, countPrivate bool) int {
```

Comment:
Is there any reason why we support counting private and non-private labels? It looks like we only need a way to count non private labels. maybe "simplify" this method and only support counting non-privates for know? (And maybe rename it to `countNonPrivateLabels` to be extra specific?)